targets/standard.props

```diff
@@ -2,7 +2,7 @@
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <ProjectDirectory Condition="'$(PathToRepo)' != ''">$(PathToRepo)</ProjectDirectory>
-    <BuildCommand>$(ProjectDirectory)/build$(ShellExtension) -$(Configuration)</BuildCommand>
+    <BuildCommand>$(ProjectDirectory)/build$(ShellExtension) -$(Configuration) -- /p:DotNetBuildFromSource=true</BuildCommand>
```

Comment:
Today this is set in the dir.props EnvironmentVariables list.  I was assuming we'd move these all to properties when we started using the official repo api stuff.  If we started adding these now, we would just need to change it again everywhere later.

Thoughts?

Reply:
Fair I can do that.